lib/proportions.js

```diff
@@ -107,16 +109,34 @@ define(['d3-interpolate', 'virtual-dom', 'filters/genericnode', 'helper'],
           return _.t('node.deactivated');
         });
 
+        nodes.forEach(function (node) {
+          if (node.flags.gateway) {
+            var mesh = node.nodeinfo.network.mesh;
+            mesh[Object.keys(mesh)[0]].interfaces.tunnel.forEach(function (mac) {
```

Comment:
"easiest" would be to rewrite the stats and also avoid some copy stuff like like all nodes to router and building a new index. (we have nodes on multiple places via setData doing similar stuff.) One data processor for all initial needed info (maybe move some tasks to yanic to reduce the work and heavy structure. e.g. in this example - i don't care for the name like mzbat or bat0)